test/client/fixtures/sandbox/node/window-test.js

```diff
@@ -11,6 +11,31 @@ test('window.onerror setter/getter', function () {
     strictEqual(getProperty(window, 'onerror'), handler);
 });
 
+module('timers');
+
+asyncTest('setTimeout script argument processing', function () {
```

Comment:
It's black box testing, right? At one moment we may decide to not override it and use another processing strategy and we will not be able to know about regression.